src/Fields/Value.php

```diff
@@ -39,13 +39,19 @@ public function raw()
 
     public function value()
     {
+        $rawValue = $this->raw;
+
         if (! $this->fieldtype) {
-            return $this->raw;
+            return $rawValue;
+        }
+
+        if (! $rawValue && $this->fieldtype->config('default')) {
+            $rawValue = $this->fieldtype->config('default');
```

Comment:
maybe `&& $defaultValue = $this->fieldtype->config('default')` then `$rawValue = $defaultValue` to save a call to the config (don't want John to get mad lol)